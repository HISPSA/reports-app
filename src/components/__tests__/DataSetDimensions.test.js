import React from 'react'
import { shallow } from 'enzyme'
import createComponentRenderer from '../../utils/test-helpers/createComponentRenderer'
import {
    DimensionDropdown,
    DataDimensionsContent,
    OriginalComponent as DataSetDimensions,
} from '../DataSetDimensions'

describe('<DimensionDropdown/>', () => {
    it('Matches the snapshot', () => {
        const props = {
            dimension: {
                id: 'test',
                displayName: 'Test',
                items: [1, 2, 3],
            },
            fullWidth: true,
            values: {
                test: 'Test',
            },
            onChange: jest.fn(),
        }
        const wrapper = shallow(<DimensionDropdown {...props} />)
        expect(wrapper).toMatchSnapshot()
    })
})
