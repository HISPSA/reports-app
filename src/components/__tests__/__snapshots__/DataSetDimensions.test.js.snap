// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DimensionDropdown/> Matches the snapshot 1`] = `
<div
  className="data-set-dimension"
  key="test"
>
  <span>
    Test
  </span>
  <DropDown
    disabled={false}
    emptyLabel="Select Option"
    fullWidth={true}
    hintText="Select Option"
    includeEmpty={true}
    menuItems={
      Array [
        1,
        2,
        3,
      ]
    }
    noOptionsLabel="-"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    value="Test"
  />
</div>
`;
